site/guides/transferring-ownership.md

```diff
@@ -3,7 +3,7 @@ outline: deep
 head:
   - - meta
     - property: og:title
-      content: How to Transfer Ownership of a Smart Account
+      content: How to Transfer Ownership of a smart account
```

Comment:
keep Smart Account capital-case here for headers and metatags

Reply:
done!